src/applications/vaos/reducers/appointments.js

```diff
@@ -58,12 +55,10 @@ export default function appointmentsReducer(state = initialState, action) {
     case FETCH_FUTURE_APPOINTMENTS_SUCCEEDED: {
       const [bookedAppointments, requests] = action.data;
 
-      const confirmedFilteredAndSorted = [...bookedAppointments]
-        .filter(filterFutureConfirmedAppointments)
-        .sort(sortFutureConfirmedAppointments);
-      const requestsFilteredAndSorted = [...requests]
-        .filter(filterRequests)
-        .sort(sortFutureRequests);
+      const confirmedFilteredAndSorted = [...bookedAppointments].sort(
+        sortFutureConfirmedAppointments,
+      );
+      const requestsFilteredAndSorted = [...requests].sort(sortFutureRequests);
```

Comment:
I moved the filtering out into selectors because the Express Care tab shows requests than the Upcoming tab doesn't, and I wanted to use the same fetched data. For consistency, I also moved the confirmed appointment filtering out into a selector.

I haven't yet moved the past appointment filtering, but I will if this approach seems like a good one.